clang/test/Driver/coverage.c

```diff
@@ -17,10 +17,6 @@
 // GCNO-LOCATION: "-coverage-notes-file={{.*}}/foo/bar.gcno"
 // GCNO-LOCATION-REL: "-coverage-notes-file={{.*}}{{/|\\\\}}foo/bar.gcno"
 
-/// GCC allows PWD to change the paths.
-// RUN: %if system-linux %{ PWD=/proc/self/cwd %clang -### -c --coverage %s -o foo/bar.o 2>&1 | FileCheck --check-prefix=PWD %s %}
-// PWD: "-coverage-notes-file=/proc/self/cwd/foo/bar.gcno" "-coverage-data-file=/proc/self/cwd/foo/bar.gcda"
```

Comment:
We should not be regressing Clang behavior like this -- at least not without wider buy-in from the Clang community via an RFC. I worry that this change will silently break build scripts relying on the current PWD behavior being consistent between Clang and GCC.

Reply:
Thank you for the timely response and for raising this important concern.

I understand the importance of maintaining consistent behavior in Clang, and I certainly do not wish to introduce changes that might silently break existing build scripts.

Could you please guide me on how to initiate the RFC process? I would like to ensure that we address this issue appropriately.

Thank you again for your insights, and I look forward to your guidance on progressing with the RFC.

Reply:
Sure! Basically, you'd write a post in https://discourse.llvm.org/ with a title along the lines of `[RFC]; stop using PWD in current_path` and some explanation of what you'd like to change, why, and what the impacts are. Folks will leave comments in the post about things you should also consider, and eventually discussion dies down and we (hopefully) have clear direction on what to do from there.

I would recommend posting this RFC in the [Clang category](https://discourse.llvm.org/c/clang/6) though it's possible the changes could impact other projects.